x-pack/plugins/ml/public/application/model_management/models_list.tsx

```diff
@@ -653,102 +611,110 @@ export const ModelsList: FC<Props> = ({
-              </EuiBadge>
-            </EuiFlexItem>
-          ))}
-        </EuiFlexGroup>
-      ),
-      'data-test-subj': 'mlModelsTableColumnType',
-    },
-    {
-      width: '10%',
       name: i18n.translate('xpack.ml.trainedModels.modelsList.stateHeader', {
         defaultMessage: 'State',
       }),
-      align: 'left',
       truncateText: false,
+      width: '150px',
       render: ({ state, downloadState }: ModelItem) => {
         const config = getModelStateColor(state);
         if (!config) return null;
 
-        const isDownloadInProgress = state === MODEL_STATE.DOWNLOADING && downloadState;
+        const isProgressbarVisible = state === MODEL_STATE.DOWNLOADING && downloadState;
 
         const label = (
-          <EuiHealth textSize={'xs'} color={config.color}>
+          <EuiText size="xs" color={config.color}>
             {config.name}
-          </EuiHealth>
+          </EuiText>
         );
 
         return (
           <EuiFlexGroup direction={'column'} gutterSize={'none'} css={{ width: '100%' }}>
-            {isDownloadInProgress ? (
+            {isProgressbarVisible ? (
               <EuiFlexItem>
                 <EuiProgress
-                  label={label}
+                  label={config.name}
                   valueText={
                     <>
-                      {((downloadState.downloaded_parts / downloadState.total_parts) * 100).toFixed(
-                        0
-                      ) + '%'}
+                      {downloadState
+                        ? (
+                            (downloadState.downloaded_parts / downloadState.total_parts) *
+                            100
+                          ).toFixed(0) + '%'
+                        : '100%'}
                     </>
                   }
-                  value={downloadState?.downloaded_parts}
-                  max={downloadState?.total_parts}
+                  value={downloadState?.downloaded_parts ?? 1}
+                  max={downloadState?.total_parts ?? 1}
                   size="xs"
                   color={config.color}
                 />
               </EuiFlexItem>
             ) : (
-              <EuiFlexItem>{label}</EuiFlexItem>
+              <EuiFlexItem grow={false}>
+                <span>{config.component ?? label}</span>
+              </EuiFlexItem>
             )}
           </EuiFlexGroup>
         );
       },
       'data-test-subj': 'mlModelsTableColumnDeploymentState',
     },
     {
-      width: '20%',
-      field: ModelsTableToConfigMapping.createdAt,
-      name: i18n.translate('xpack.ml.trainedModels.modelsList.createdAtHeader', {
-        defaultMessage: 'Created at',
-      }),
-      dataType: 'date',
-      render: (v: number) => dateFormatter(v),
-      sortable: true,
-      'data-test-subj': 'mlModelsTableColumnCreatedAt',
-    },
-    {
-      width: '15%',
       name: i18n.translate('xpack.ml.trainedModels.modelsList.actionsHeader', {
         defaultMessage: 'Actions',
```

Comment:
Is there an easy way to pick which icons are displayed when there are two actions displayed in the column? The current setup shows the 'Test' action, but now I see the Data Drift action for DFA models. 'Test' is probably more commonly used. If not easy, leave it as it is.

Reply:
I made the Test action primary in 6fb1c5ee5c8b8ffd1069a037b6913135b29608b3